js/test/Ice/optional/InitialI.ts

```diff
@@ -59,209 +59,203 @@ export class InitialI extends Test.Initial {
+    opMyEnum(p1: Test.MyEnum | undefined, current: Ice.Current): [Test.MyEnum | undefined, Test.MyEnum | undefined] {
         return [p1, p1];
     }
 
     opSmallStruct(
-        p1: Ice.Optional<Test.SmallStruct>,
+        p1: Test.SmallStruct | undefined,
         current: Ice.Current,
-    ): [Ice.Optional<Test.SmallStruct>, Ice.Optional<Test.SmallStruct>] {
+    ): [Test.SmallStruct | undefined, Test.SmallStruct | undefined] {
         return [p1, p1];
     }
 
     opFixedStruct(
-        p1: Ice.Optional<Test.FixedStruct>,
+        p1: Test.FixedStruct | undefined,
         current: Ice.Current,
-    ): [Ice.Optional<Test.FixedStruct>, Ice.Optional<Test.FixedStruct>] {
+    ): [Test.FixedStruct | undefined, Test.FixedStruct | undefined] {
         return [p1, p1];
     }
 
     opVarStruct(
-        p1: Ice.Optional<Test.VarStruct>,
+        p1: Test.VarStruct | undefined,
         current: Ice.Current,
-    ): [Ice.Optional<Test.VarStruct>, Ice.Optional<Test.VarStruct>] {
+    ): [Test.VarStruct | undefined, Test.VarStruct | undefined] {
         return [p1, p1];
     }
 
     opOneOptional(
-        p1: Ice.Nullable<Test.OneOptional>,
+        p1: Test.OneOptional | null,
         current: Ice.Current,
-    ): [Ice.Nullable<Test.OneOptional>, Ice.Nullable<Test.OneOptional>] {
+    ): [Test.OneOptional | null, Test.OneOptional | null] {
         return [p1, p1];
     }
 
     opMyInterfaceProxy(
-        p1: Ice.Optional<Ice.Nullable<Test.MyInterfacePrx>>,
+        p1: Test.MyInterfacePrx | null | undefined,
         current: Ice.Current,
-    ): [Ice.Optional<Ice.Nullable<Test.MyInterfacePrx>>, Ice.Optional<Ice.Nullable<Test.MyInterfacePrx>>] {
+    ): [Test.MyInterfacePrx | null | undefined, Test.MyInterfacePrx | null | undefined] {
         return [p1, p1];
     }
 
-    opByteSeq(
-        p1: Ice.Optional<Uint8Array>,
-        current: Ice.Current,
-    ): [Ice.Optional<Uint8Array>, Ice.Optional<Uint8Array>] {
+    opByteSeq(p1: Uint8Array | undefined, current: Ice.Current): [Uint8Array | undefined, Uint8Array | undefined] {
         return [p1, p1];
     }
 
-    opBoolSeq(p1: Ice.Optional<boolean[]>, current: Ice.Current): [Ice.Optional<boolean[]>, Ice.Optional<boolean[]>] {
+    opBoolSeq(p1: boolean[] | undefined, current: Ice.Current): [boolean[] | undefined, boolean[] | undefined] {
         return [p1, p1];
     }
 
-    opShortSeq(p1: Ice.Optional<number[]>, current: Ice.Current): [Ice.Optional<number[]>, Ice.Optional<number[]>] {
+    opShortSeq(p1: number[] | undefined, current: Ice.Current): [number[] | undefined, number[] | undefined] {
         return [p1, p1];
     }
 
-    opIntSeq(p1: Ice.Optional<number[]>, current: Ice.Current): [Ice.Optional<number[]>, Ice.Optional<number[]>] {
+    opIntSeq(p1: number[] | undefined, current: Ice.Current): [number[] | undefined, number[] | undefined] {
         return [p1, p1];
     }
 
     opLongSeq(
-        p1: Ice.Optional<Ice.Long[]>,
```

Comment:
Also here, `Optional` changed to `| null | undefined` instead of just `| undefined`.